js/main.js

```diff
@@ -76,35 +71,27 @@ function renderEntry(entry) {
 }
 
 function dataLoop() {
-  if (isOn === true) {
-    for (var i = 0; i < data.entries.length; i++) {
-      var entriesDOM = renderEntry(data.entries[i]);
 
-      hiddenDiv.appendChild(entriesDOM);
-    }
+  for (var i = 0; i < data.entries.length; i++) {
+    var entriesDOM = renderEntry(data.entries[i]);
+
+    hiddenDiv.appendChild(entriesDOM);
   }
 
   viewSwap(data.view);
   if (data.entries) {
```

Comment:
This conditional is still not necessary. Even if the array was empty, this code would still run. An empty array is truthy, not falsy, since it has a memory space created for it, even if there is nothing saved there yet. You still need the code inside of it, though.